static/app/components/replays/header/detailsPageBreadcrumbs.tsx

```diff
@@ -42,10 +42,19 @@ function DetailsPageBreadcrumbs({orgSlug, replayRecord}: Props) {
         {
           label: (
             <Fragment>
-              {<BaseBadge displayName={labelTitle} project={project} avatarSize={16} />}
+              {
+                <BaseBadge
+                  displayName={project?.slug}
+                  project={project}
+                  avatarSize={16}
+                />
```

Comment:
Oh just noticed, we could use the `<ProjectBadge>` and maybe even for the `tableCell` -- Looking at the props for that it looks like it auto links to the project details, which we can disable with `disableLink`

Reply:
Actually... maybe Avatar is what we want?